tools/vf-component-library/src/site/about/consultation.njk

```diff
@@ -115,18 +115,52 @@ At a high level it [follows this flow diagram](https://coggle.it/diagram/V0hkiP9
 
 ### Who
 
-- VF core maintainers (Ken, Stu)
-- EMBL Communications (notably Digital and Design)
-- EMBL-EBI Web Development and UX
-- Nominated service representatives (and those interested)
+- VF core maintainers
+- EMBL Communications' Digital and Design teams
+- EMBL-EBI Web Development, UX and Communications teams
+- Nominated service representatives
   - Relationship with EMBL-EBI Web Guidelines Committee
 - The requestor(s) of a new component or change
 
-#### What if...
+### When
 
-My idea gets rejected? The agreed implemention doesn't cover my needs?
+In addition to the quarterly consultation, we meet on regular basis in three channels:
 
-Even if your need can't be acconomidated in the global VF repositiory, You can still [create and use a custom component in your local project]({{ '/building' | url }}). You can also share that component on npm for others to use.
+- Design consultation, every other Monday
+- UX consultation, every other Tuesday
+- Technical consultation, monthly on the first Thursday
+
+These form initial discussion and debate on new ideas before they are further tested and actioned on the GitHub issue queue.
+
+{% endmarkdown %}
+
+</div>
+</section>
+
+<section class="embl-grid embl-grid--has-centered-content">
+{% render '@vf-section-header', {
+  "section_title": "How to propose",
+  "id": "propose",
+  "vf_section__content": ""
+} %}
+<div class="vf-content">
+
+{% markdown %}
+
+If you have an idea you'd like to suggest, the best place to begin is to [raise the topic on Slack](https://visual-framework.slack.com/join/shared_invite/enQtNDAxNzY0NDg4NTY0LWFhMjEwNGY3ZTk3NWYxNWVjOWQ1ZWE4YjViZmY1YjBkMDQxMTNlNjQ0N2ZiMTQ1ZTZiMGM4NjU5Y2E0MjM3ZGQ#/).
```

Comment:
do we want to include something about the GitHub discussions tab here?

Reply:
👍 added it